tests/back/Pim/Enrichment/EndToEnd/Product/ProductModel/ExternalApi/PartialUpdateProductModelEndToEnd.php

```diff
@@ -519,6 +519,30 @@ public function testUpdateRootProductModel()
         $this->assertEventCount(1, ProductModelUpdated::class);
     }
 
+    public function testUpdateCaseInsensitiveProductModel()
+    {
+        $client = $this->createAuthenticatedClient();
+
+        $data =
+            <<<JSON
+{
```

Comment:
```suggestion
        $data =
<<<JSON
{
```
Except for one test in this file, the standard is this. In the only one that differ and in other files, I found this :
```
        $data = <<<JSON
{
```